assets/blocks/frontend.js

```diff
@@ -1,14 +1,9 @@
-/**
- * WordPress dependencies
- */
-import domReady from '@wordpress/dom-ready';
-
 /**
  * Internal dependencies
  */
 import '../js/sensei-modal';
 
-domReady( () => {
+window.onload = () => {
```

Comment:
We changed the event because we need to make sure that the content is ready to calculate the height.

Reply:
Makes sense! But it should be `window.addEventListener( "load", () => {`, using `onload =` easily breaks if another script is doing it and they ovewrite eachother.

Reply:
Oops! Good catch! Commit updated: https://github.com/Automattic/sensei/pull/4677/commits/6789156ffef2c674455731a71d5e8cc2e41ff77e